packages/connect-common/src/systemInfo.ts

```diff
@@ -42,7 +51,7 @@ export const getSystemInfo = (supportedBrowsers: {
         ? supportedBrowser.version > parseInt(browserVersion, 10)
         : false;
     const mobile = getDeviceType() === 'mobile';
-    const supportedMobile = mobile ? typeof navigator.usb !== 'undefined' : true;
+    const supportedMobile = mobile ? 'usb' in navigator : true;
```

Comment:
I imagine this is right, but I am trying to understand why when `mobile` is false then `supportedMobile` is true.
:exploding_head:

Reply:
if device type is detected as mobile, we set supportedMobile field (which means is popup supported on mobile?) based on presence of navigator. and regarding your question - it is probably because of the line below 
```
    const supported = !!(supportedBrowser && !outdatedBrowser && supportedMobile);

```